README

```diff
@@ -163,6 +163,16 @@ Feature roadmap (the features I want to implement in a subsequent release)
 
 ---------------------------------------------------------
 
+---------------------------------------------------------
```

Comment:
Please make this change into the resources/readme.src file.  Then from the resources directory, run "make README".  The copy the resulting README to the repo root.  Commit README, resources/README, and resources/readme.src.

Reply:
Done, however I noticed that you cannot run `make README` without first doing a `make` in root directory. And if you do `make README` after that it tells you that is already up to date